tools/mason/MasonHelp.chpl

```diff
@@ -403,3 +405,43 @@ proc masonSystemPcHelp() {
 }
 
 
+proc masonPublishHelp(){
+  writeln("Publish a package to the mason-registry repository");
+  writeln();
+  writeln("Usage:");
+  writeln("    mason publish [options]");
+  writeln();
+  writeln("Options:");
+  writeln("    -h, --help                  Display this message");
+  writeln(); 
+}
+
+
+proc masonCleanHelp(){
+  writeln("Uninstall mason from a mason package");
```

Comment:
This doesn't sound right. It cleans the `target/` directory of the mason package, not mason itself.